### PR TITLE
Update deployment artifacts

### PR DESCRIPTION
**Pull request by sanethmaduranga**

### CODE REVIEW

File: roles/apim-devportal/templates/carbon-home/repository/conf/deployment.toml.j2 [hunk 1]
```diff
@@ -113,7 +113,7 @@ name="userPostSelfRegistration"
 subscriptions=["POST_ADD_USER"]
 
 [apim.throttling]
-service_url = "{{ api_devportal_server_url }}"
+service_url = "{{ traffic_manager_host_url }}/services/"
```
Comment (VimukthiPerera) [inline roles/apim-devportal/templates/carbon-home/repository/conf/deployment.toml.j2:116]: Is `traffic_manager_host_url` initialised somewhere?

**Review by VimukthiPerera — CHANGES_REQUESTED** [review]

File: roles/apim-devportal/templates/carbon-home/repository/conf/deployment.toml.j2 (hunk 1 above)
Comment (sanethmaduranga) [reply roles/apim-devportal/templates/carbon-home/repository/conf/deployment.toml.j2:116]: It is in the 'dev/group_vars/apim.yml' file.

**Comment by VimukthiPerera** [comment]
@sanethmaduranga Please refer to https://chris.beams.io/posts/git-commit/#capitalize and change the commit message.

File: roles/apim-devportal/templates/carbon-home/repository/conf/deployment.toml.j2 (hunk 1 above)
Comment (VimukthiPerera) [reply roles/apim-devportal/templates/carbon-home/repository/conf/deployment.toml.j2:116]: ACK.

File: dev/group_vars/apim-analytics.yml [hunk 2]
```diff
@@ -88,7 +88,7 @@ message_tracing_db_password: wso2carbon
 message_tracing_db_driver: org.h2.Driver
 message_tracing_db_test_query: SELECT 1
 
-apim_analytics_db_url: jdbc:h2:${sys:carbon.home}/wso2/worker/database/WSO2AM_STATS_DB;AUTO_SERVER=TRUE
+apim_analytics_db_url: jdbc:h2:${sys:carbon.home}/../../apim-analytics-worker/wso2am-analytics-3.2.0/wso2/database/WSO2AM_STATS_DB;AUTO_SERVER=TRUE
```
Comment (Savidude) [inline dev/group_vars/apim-analytics.yml:91]: Shall we parameterize the version in `wso2am-analytics-3.2.0` here? Could be an area someone might miss when modifying the Ansible resources for a new release.

Comment (sanethmaduranga) [reply dev/group_vars/apim-analytics.yml]: Fixed with e79968f8fe8f2c69db8216cb1d4d5137559e4b3b

**Review by VimukthiPerera — APPROVED** [review]

**Review by Savidude — APPROVED** [review]

**Pull request merged**